container_pipeline/workers/scanners/scan.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+
+import docker
+import json
+import logging
+import os
+import shutil
+import subprocess
+import config
+import hashlib
+
+from Atomic import Atomic, mount
+
+from container_pipeline.utils import Build, get_job_name
+from container_pipeline.lib.log import load_logger
+from container_pipeline.workers.base import BaseWorker
+from container_pipeline.workers.scanners.runner import ScannerRunner
+
+class ScanWorker(BaseWorker):
+    """Scan worker"""
+
+    def handle_job(self, job):
+        """
+        This scans the images for the job requests in start_scan tube.
+        this cals the ScannerRunner for performing the scan work
+        """
+
+        debug_logs_file = os.path.join(
+            job_info["logs_dir"], "service_debug.log")
+        dfh = config.DynamicFileHandler(logger, debug_logs_file)
+
+        self.logger.info('Got job: %s' % job)
+        scan_runner_obj = ScannerRunner(job)
+        status, scanners_data = scan_runner_obj.scan()
+        if not status:
+            logger.critical(
+                "Failed to run scanners on image under test, moving on!"
+            )
+        else:
+            self.logger.info(str(scanners_data))
+
+        # Remove the msg and logs from the job_info as they are not
+        # needed now
+        scanners_data.pop("msg", None)
+        scanners_data.pop("logs", None)
+        scanners_data.pop("scan_results", None)
```

Comment:
but scanners_data was never passed to notify_worker or it is being checked in some other way. I believe job_info was not having these details. https://github.com/CentOS/container-pipeline-service/blob/master/beanstalk_worker/worker_start_scan.py#L602.
How do you think? @navidshaikh